lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -78,6 +78,45 @@ def approved?
           end
         end
 
+        def merge_feature_branch(feature_branch_name, options = {})
```

Comment:
We are currently using [method_missing](https://github.com/simonzhu24/gitreflow/blob/sz-issue/149-Fix-Commit-Message-to-merge-instead-of-close/lib/git_reflow/git_server/pull_request.rb#L136) to delegate methods to the `source_object`.  So anything you can call on a a `github_api` PR object, you can call in this context.  Reference the existing `deliver` method for how we're passing those options.